types/expo/index.d.ts

```diff
@@ -269,14 +340,14 @@ export namespace Audio {
         canRecord: false,
         isDoneRecording: false
     } | {
-        canRecord: true,
```

Comment:
Please remove these indentation changes.

Reply:
Will do - they were unintended, I thought I'd turned off all autoformatting but obviously not.